### PR TITLE
Doc fixes2

### LINKED ISSUE #526: documentation fixes
some minor points regarding the documentation

- the `Q_` docstring seems to have some errors in the examples formatting: https://weldx.readthedocs.io/en/latest/_autosummary/weldx.Q_.html
- `Time` is missing at the user API section (should be next to `Q_` I guess)
- can we get the objects listed in the `constants` module? https://weldx.readthedocs.io/en/latest/_autosummary/weldx.constants.html (not sure how this works, do we add docstrings?)
- move `GEOMETRY` section to the end of the user API documentation (imo)

### PR DESCRIPTION
**Pull request by marscher**
## Changes

minor doc fixes

## Related Issues

Closes #526

## Checks

- [x] updated doc/

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -273,7 +273,7 @@ def _prevent_sphinx_circular_imports_bug():
     "matplotlib": ("https://matplotlib.org", None),
     # "dask": ("https://docs.dask.org/en/latest", None),
     # "numba": ("https://numba.pydata.org/numba-doc/latest", None),
-    "pint": ("https://pint.readthedocs.io/en/stable", None),
+    "pint": ("https://pint.readthedocs.io/en/latest", None),
```
Comment (CagtayFabry) [inline doc/conf.py:276]: I had hoped this would include your fix in the `pint` docs but somehow it doesn't 😞  @marscher

**Comment by CagtayFabry** [comment]
I'm fine with leaving `U_` out of the documentation for now since it doesn't get any real use so far in user facing code

File: doc/conf.py (hunk 1 above)
Comment (marscher) [reply doc/conf.py:276]: They merged it quite some time ago, but it hasn't been released yet, right? I'd guess latest is just link to the latest release, not unreleased main branch stuff :)

**Comment by marscher** [comment]
I guess that is fine. We never use `Unit` in an example or tutorial.

File: doc/conf.py (hunk 1 above)
Comment (CagtayFabry) [reply doc/conf.py:276]: I think `stable` is the current release and `latest` is actually the main branch (same as for weldx or asdf etc.)

The fix is in there but still doesn't work somehow..
https://pint.readthedocs.io/en/latest/developers_reference.html?highlight=to_compact#pint.Quantity.to_compact

**Comment by CagtayFabry** [comment]
> I really do not understand, why Sphinx cannot find the TimeDependent class in the Time docstring.

Don't know, in that case just double escape it to disable linking and we can look into it another time

**Comment by CagtayFabry** [comment]
merci

**Pull request merged**